### DISCUSSION TITLE
Positron console customization: Ability to distinguish prompts color sent to the console and the output received

### DISCUSSION THREAD

**Discussion by anbrav0** [posit-dev/positron#5069] [Feedback]
Hello Positron team and Positron discussion board!

Thank you for working on this project. I've been trying out Positron since the public beta and noticed there is some Positron specific JSON settings that can be called to customize certain this in the console (see below) 

<img width="747" alt="image" src="https://github.com/user-attachments/assets/93251e26-9455-4364-976b-868c52b11dc1">

I am curious if there is a Positron command in the settings JSON file, or in the Positron environment that i can directly call to change the color differentiation between sending a function to the console and the output. For example, in RStudios when I call a data frame called "raw_penguins" and select a variable, the output of that call shows a tibble but the text out that output is white, while sending the function call to the console it appears orange (as the default of one of the themes in RStudios):

<img width="1028" alt="image" src="https://github.com/user-attachments/assets/e0be1b0e-d808-4528-b3f5-19713c93b616">

In Positron, when i do the same thing the code sent and the output received from the console remain the same color:
<img width="697" alt="image" src="https://github.com/user-attachments/assets/8b9d57c2-6702-4196-bd79-7d2090438d0e">

that differentiation of color is something that helps in my data science work flows sometimes when I need to check back the console to quickly look over my history and find that particular call later on.

In the first picture above you'll see i call `"positronConsole.foreground": "#dec5f0dc",` for some minimal color change from the default, so I am curious if I can replicate something similar from RStudios, it possible it's something that might already be there and I for some reason cannot find it. :) 

thank you for your time!

**Comment by testlabauto** [comment]
Thank you for using Positron.  We appreciate your suggestion and will properly prioritize it.

Would you mind filing this a feature request?  Just follow the [New Issue](https://github.com/posit-dev/positron/issues) flow and prefix your issue name with "Feature Request".

**Reply by anbrav0** [reply]
Hi @testlabauto! thanks! opened at #5115
